tests/suites/0_stateless/03_dml/03_0016_insert_into_values_v2.sql

```diff
@@ -0,0 +1,19 @@
+set enable_planner_v2 = 1;
+
+DROP DATABASE IF EXISTS db1;
+CREATE DATABASE db1;
+USE db1;
+
+CREATE TABLE IF NOT EXISTS t1(a Int8 null, b UInt32 null, c DateTime null, d String null) Engine = Memory;
```

Comment:
Hmm, don't use `Engine = Memory` for the test, use `Fuse` engine by default.

Reply:
Ok, I just copy the original test file. The original test file has changed in #5530